src/main/kotlin/mate/academy/LetterRemover.kt

```diff
@@ -1,5 +1,7 @@
 package mate.academy
 
-class LetterRemover {
+object LetterRemover {
```

Comment:
why object? you can use just fun

Reply:
When I tried this in the 1st pullrequest, the build with the command "./gradlew build" was failed: